sync3c.go

```diff
@@ -67,6 +68,7 @@ func main() {
 	flag.StringVar(&name, "name", "", "download media of a specific conference only (e.g. '33c3')")
 	flag.StringVar(&downloadPath, "destination", "./downloads/", "where to store downloaded media")
 	flag.StringVar(&language, "language", "", "preferred language if available (eng, deu or fra)")
+	flag.StringVar(&source, "source", "", "source of conferences (e.g. 'media.ccc.de', 'media.freifunk.net'")
```

Comment:
If you define the default to be `media.ccc.de` here (third parameter), we could avoid the check below (line 90-92).